pages/api/helpers/api/error-handler.ts

```diff
@@ -7,6 +7,7 @@ function errorHandler (
   res: NextResponse
 ): void {
   if (typeof(err) === 'string') {
+    console.log(err);
```

Comment:
Could we use `console.error()`? https://www.w3schools.com/jsref/met_console_error.asp.

In k8s world, containers that wish to report an error should write to the error interface. In a managed k8s cluster like EKS, the cloud provider typically has a modified version of k8s' logs and error collection libraries and will redirect logs to their cloud log and offerings.

In the same fashion, using `console.log()` should redirect logs to k8s' log collection mechanism.